extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -473,21 +420,11 @@ export class ConsentPolicyInstance {
   }
 
   /**
-   * Returns the current consent string info
-   * @return {?string}
-   */
-  getConsentString() {
-    // Return a string because we only support one consent instance now.
-    // And decide not to support multiple consent instances given CMP approach
-    return this.consentString_;
-  }
-
-  /**
-   * Returns whether the current consent policy state should block
+   * Returns whether the current consent policy state should be unblocked
    * Caller need to make sure that policy status has resolved
    * @return {boolean}
    */
-  shouldBlock() {
+  shouldEnable() {
```

Comment:
As we discussed offline, let's just keep this as unblock, my apologies 😢

Reply:
renamed back to `shouldUnblock`